api/api.go

```diff
@@ -195,6 +201,9 @@ func (api *Server) GetBlockMetas(ctx context.Context, in *iotexapi.GetBlockMetas
 
 // GetChainMeta returns blockchain metadata
 func (api *Server) GetChainMeta(ctx context.Context, in *iotexapi.GetChainMetaRequest) (*iotexapi.GetChainMetaResponse, error) {
+	if !api.hasActionIndex {
```

Comment:
for chain meta, if hasActionIndex is false, can we still return the result normally, but set total action number to 0

Reply:
when https://github.com/iotexproject/iotex-core/pull/1235 merged,GetChainMeta will not need action index